tools/BUILD.bazel

```diff
@@ -0,0 +1,60 @@
+# Note: this isn't actually generated at build time, but you can run this rule
```

Comment:
Can you comment why this isn't generated at build time? Limitation or intentional decision (why)?

Reply:
done